subsystems/DrivetrainBase.java

```diff
@@ -1,5 +1,65 @@
 package org.frc5587.lib.subsystems;
 
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.wpilibj.drive.DifferentialDrive;
+import edu.wpi.first.wpilibj.SpeedControllerGroup;
+
 public class DrivetrainBase {
-    
-}
+    // create leader and follower motors for the drivetrain
+    private final WPI_TalonFX leftLeader = new WPI_TalonFX(0);
+    private final WPI_TalonFX rightLeader = new WPI_TalonFX(1);
```

Comment:
In terms of setting the motor IDs, you shouldn't declare absolute values here. Instead, assign variables within the arguments of the constructor of the class, which then get declared in the constructor.